src/components/loaders/TrackedEntityLoader.js

```diff
@@ -1,12 +1,14 @@
+import { useConfig } from '@dhis2/app-runtime'
 import PropTypes from 'prop-types'
 import { useEffect } from 'react'
 import trackedEntityLoader from '../../loaders/trackedEntityLoader.js'
 import useLoaderAlerts from './useLoaderAlerts.js'
 
 const TrackedEntityLoader = ({ config, onLoad, loaderAlertAction }) => {
     const { showAlerts } = useLoaderAlerts(loaderAlertAction)
+    const { serverVersion } = useConfig()
     useEffect(() => {
-        trackedEntityLoader(config).then((result) => {
+        trackedEntityLoader(config, serverVersion).then((result) => {
```

Comment:
`serverVersion` needs to be added as a dependency to this effect

Reply:
Addressed here: https://github.com/dhis2/maps-app/pull/3346/commits/e7db4d71c661a9970786e5d35b58194ba1b093be